scm/client.go

```diff
@@ -86,6 +86,9 @@ type (
 		// Base URL for API requests.
 		BaseURL *url.URL
 
+		// project ID only used for azure devops.
+		AzureRepoID string
```

Comment:
this can be moved to `wrapper` so that we don't need to leak any driver specific details into the main scm package